src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt

```diff
@@ -24,31 +24,37 @@ val controlModeChooser = SendableChooser<ControlMode>().apply {
 val controlMode get() = controlModeChooser.selected ?: ControlMode.CONTROLLER
 
 val primaryJoystick by lazy { Joystick(Operator.PRIMARY_STICK) }
+val secondaryJoystick by lazy { Joystick(Operator.SECONDARY_STICK) }
 val primaryController by lazy { XboxController(Operator.PRIMARY_CONTROLLER) }
 
 fun CoroutineScope.getInputs() {
     SmartDashboard.putData("Control Mode", controlModeChooser);
 
-    { primaryJoystick.getRawButton(3) }.watch {
+    { primaryJoystick.getRawButton(Operator.CUBEINTAKE_BUTTON) }.watch {
         whileTrue {
             intakeCubes()
         }
     };
-    { primaryJoystick.getRawButton(11) }.watch {
+    { secondaryJoystick.getRawButton(Operator.CUBEOUTTAKE_BUTTON) }.watch {
         whileTrue {
             outtakeCubes()
         }
     };
-    { primaryJoystick.getRawButton(Operator.TUBINTAKE_IN_BUTTON) }.watch() {
+    { secondaryJoystick.getRawButton(Operator.TUBINTAKE_IN_BUTTON) }.watch {
         whileTrue {
             println("Intake should be spinning in")
             tubIntake()
         }
     };
-    { primaryJoystick.getRawButton(Operator.TUBINTAKE_OUT_BUTTON) }.watch() {
+    { secondaryJoystick.getRawButton(Operator.TUBINTAKE_OUT_BUTTON) }.watch {
         whileTrue {
             println("Outtake should be running")
             tubOuttake()
         }
+    };
```

Comment:
Are the semicolons here necessary? (And at the end of previous lines)

Reply:
I think so, but I'm not sure why. It yells at me if I don't add any semicolons.

Reply:
Huh that's super strange, guess it's just Kotlin oddities!